community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/BatchingTransactionAppender.java

```diff
@@ -93,11 +93,6 @@ public void start() throws Throwable
     @Override
     public long append( TransactionToApply batch, LogAppendEvent logAppendEvent ) throws IOException
     {
-        // We put log rotation check outside the private append method since it must happen before
```

Comment:
So true, **outside** the private append method is not the same as **before** the private append method, my bad.